source/index.html.md

```diff
@@ -270,20 +287,23 @@ A successful request returns the HTTP `201 Created` status code with the followi
 | ---| ---| ---| --- |
 | user\_id | string | QEaakM0qumTJd5tj | The Betable user\_id as returned from the [Create User](#create-user) method call. |
 
+<aside class="success">
+The <code>user_id</code> returned should be used in subsequent API calls where a <code>user_id</code> is required as part of the URI.
+</aside>
 
 | HTTP Status Code | Error Description |
 | ---| --- |
 | `201 Created` | Success
-| `208 Already Reported` | Success, if the `external_user_id` was already processed |
+| `208 Already Reported` | Success, the user with `external_user_id` was already created |
```

Comment:
I tried to fix these quickly the wording seemed odd, what does it mean to "process the external_user_id"?

Reply:
Created seems good.  But a new user with the same external_user_id returns both 208 and 409 in this chart.

Reply:
Yeah I called this problem out later in the doc that we might have to document both cases in all create calls where a partner specifies an external id

208 - when a partner uses idempotency header and retries a request
409 - when a partner either does not use idempotency or submits a new request with the same external id